### ISSUE TITLE
Final QA of site

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#70]
NOTE: not ready to start until March 13th at the earliest. [Staging branch here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/) for QA. 

**Desktop**
- [x] Review content for spelling, grammatical errors, and accuracy
- [x] Ensure all mentions of buyers and sellers guide have the apostrophe after `buyers'` and `sellers'`
- [x] Test the GSA.gov menu
- [x] Make sure all links work
- [x] check to make sure external links have appropriate external link icon (gsa.gov proper (inside the CMS) is internal, everything else is external)*

**Mobile**
- [x] Test the GSA.gov menu
- [x] Review all pages on mobile device and indicate if anything appears wonky / distorted 

Record all findings in the [OASIS+ content updates](https://docs.google.com/document/d/1tnZwnh90D4EoJkhPz53H50q1aAc0JGDA_u-cIf_IaXw/edit) document

* Only links to gsa.gov/ sites within the CMP are treated as internal (i.e. smartpay.gsa.gov, interact.gsa.gov, sam.gov are all external)

**Comment by vickimcfadden** [comment]
@rosamundtgov would appreciate anything else you want to specifically call out for attention in the QA review of the site

**Comment by vickimcfadden** [comment]
i did desktop and mobile review, findings captured in the [OASIS+ content updates](https://docs.google.com/document/d/1tnZwnh90D4EoJkhPz53H50q1aAc0JGDA_u-cIf_IaXw/edit#heading=h.6fhubw4mqarb) doc.

**Comment by JessicaMarine1** [comment]
@vickimcfadden do I still need to do a final QA review? I'd set aside some time tomorrow for this but see that you already did one pass.

**Comment by vickimcfadden** [comment]
Yes please! Even though I haven't been as close as Kristen and Rosamund, I still have been "in" this work for months. Would appreciate a fresh perspective!

**Comment by JessicaMarine1** [comment]
I finished my review and provided feedback [here](https://docs.google.com/document/d/1tnZwnh90D4EoJkhPz53H50q1aAc0JGDA_u-cIf_IaXw/edit#heading=h.hec3ctstjx3c). Please let me know if you have any questions.

**Comment by klohman** [comment]
@ekidenda @PowderD This item has been completed and can be moved to Done.
